optimum_benchmark/task_utils.py

```diff
@@ -75,6 +75,9 @@ def infer_library_from_model_name_or_path(model_name_or_path: str, revision: Opt
         if inferred_library_name is None:
             raise RuntimeError(f"Could not infer library name from repo {model_name_or_path}.")
 
+        if "gguf" in model_name_or_path.lower():
+            inferred_library_name = "llama_cpp"
+
```

Comment:
remove. at this point the code will have errored and the hub already tags repos with gguf files https://huggingface.co/models?library=gguf

Reply:
Thanks for the remark about tags i did not know this feature. I am now using it to setup the inferred_library_name properly.